packages/components/src/components/primitives/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from 'react';
+import { IconButton as ChakraIconButton, useMultiStyleConfig } from '@chakra-ui/react';
+import { IconButtonProps } from './types';
+
+export const IconButton: FC<IconButtonProps> = ({ ...props }) => {
```

Comment:
no need to destructure props

```suggestion
export const IconButton: FC<IconButtonProps> = (props) => {
```

Reply:
by overriding the `rounded` chakra prop (which is not a boolean in native Chakra) now it is required to extract it from the props destructured object so `props` can be spread directly into the chakra's `IconButton` component